rocky/tools/models.py

```diff
@@ -160,6 +163,8 @@ def pre_create(cls, sender, instance, *args, **kwargs):
 
         try:
             octopoes_client.create_node()
+            valid_time = datetime.datetime.now(datetime.timezone.utc)
+            octopoes_client.save_declaration(Declaration(ooi=Network(name="internet"), valid_time=valid_time))
```

Comment:
I think this needs better exception handling. The problem here is that if the declaration save fails while the organization creation succeeds, it will still revert the process, leading to the deletion of the organization in the KATalogus, while leaving the organization created in the XTDB multinode.

Reply:
Maybe it makes more sense to move this to a `post_save` signal instead of this `pre_save` signal

Reply:
👍 Done